hooks/nrpe_helpers.py

```diff
@@ -888,3 +881,68 @@ def set_netlinks_error():
     """Set the flag indicating a netlinks related error."""
     global NETLINKS_ERROR
     NETLINKS_ERROR = True
+
+
+def _get_cis_audit_check():
+    """Get the CIS audit step for nrpe.
+
+    When CIS audit is disabled no step is necessary.
+    """
+    if not hookenv.config("cis_audit_enabled"):
+        return {}
+
+    cis_cmd = cis_cmd_params(include_score=True)
+
+    cis_audit_check = {
+        "description": "Check CIS audit",
+        "cmd_name": "check_cis_audit",
+        "cmd_exec": local_plugin_dir + "check_cis_audit.py",
+        "cmd_params": cis_cmd,
+    }
+    return cis_audit_check
+
+
+def is_cis_misconfigured():
```

Comment:
Nonblocking, but I realise that this is opposite to usual convention - normally we'd have a function like `is_cis_valid` that returns True if the config **is** valid.